sympy/functions/elementary/tests/test_exponential.py

```diff
@@ -452,3 +453,12 @@ def test_issue_8866():
     b2 = log(exp(y), exp(5), evaluate=False)
     assert simplify(log(l1, b1)) == simplify(log(l2, b2))
     assert expand_log(log(l1, b1)) == expand_log(log(l2, b2))
+
+
+def test_issue_9116():
+    n = Symbol('n', positive=True, integer=True)
+    p = Symbol('p', prime=True)
```

Comment:
This is redundant. You are simply testing that a prime is recognized to be 2 or greater. Testing `n` should be sufficient.